### ISSUE TITLE
Failed to run ./configure,  syntax error near unexpected token `LIBTIRPC,'

### ISSUE THREAD

**Issue by Cypresslin** [linux-test-project/ltp#649]
When trying to build the LTP test suite, the ./configure will failed with:

    ./configure: line 6787: syntax error near unexpected token `LIBTIRPC,'
    ./configure: line 6787: `		PKG_CHECK_MODULES(LIBTIRPC, libtirpc,'

A bisect shows c7d2a745 is the first bad commit. It will work If you reset HEAD to c7d2a745^

Environment: Ubuntu 16.04 (4.4.0-174-generic), libtirpc-dev (0.2.5-1ubuntu0.1)

**Comment by Cypresslin** [comment]
It turns out that we need to install pkg-config package, sorry for the noise, I will close this issue here.
Thanks!

**Comment by pevik** [comment]
Yep. The quickest way (for next time) is to check changes in `travis/*.sh` scripts.

**Comment by pevik** [comment]
BTW I also got into this trap when writing c7d2a74. I took me some time to find, that `/usr/share/aclocal/pkg.m4` is not in autotools, but external dependency [provided by pkg-config](https://gitlab.freedesktop.org/pkg-config/pkg-config/-/blob/master/pkg.m4.in). I planned to stress in that commit this dependency, but forgot on it (it's actually in my WIP patch for netlink route tests, which also uses pkg-config autotools detection, which I thought it'd got merged before this).
